src/components/FooterSection.vue

```diff
@@ -7,15 +7,12 @@
             href="https://creativecommons.org"
             class="main-logo margin-bottom-bigger has-text-white"
           >
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              preserveAspectRatio="xMidYMid meet"
-              viewBox="0 0 304 73"
-            >
-              <use
-                href="~@creativecommons/vocabulary/assets/logos/cc/logomark.svg#logomark"
-              />
-            </svg>
+            <img
+              src="@creativecommons/vocabulary/assets/logos/cc/logomark.svg"
```

Comment:
We are preparing to deprecate the `@creativecommons/vocabulary` package. Use the `cc-assets` package instead.

Reply:
Oh yeah let me correct that. Must have been a typo.